src/main/java/com/tvtracker/enterprise/service/UserAccountServiceStub.java

```diff
@@ -56,8 +56,9 @@ public UserAccount createUserAccount(UserAccount userAccount) throws Exception {
      */
     @Override
     public UserAccount fetchUserAccount(String username) throws Exception {
-        if(username == null)
+        if(username == null) {
```

Comment:
Adding brackets to IF statements is considered best practice based on the source below. 

Source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3

Reply:
Generally a good idea.  Makes the source code more readable; less error prone.